models/yield_yak/avalanche_c/yield_yak_avalanche_c_trades.sql

```diff
@@ -52,31 +54,31 @@ SELECT
     END as token_pair, 
     dexs.token_bought_amount_raw / power(10, erc20a.decimals) as token_bought_amount, 
     dexs.token_sold_amount_raw / power(10, erc20b.decimals) as token_sold_amount, 
-    CAST(dexs.token_bought_amount_raw AS DECIMAL(38,0)) AS token_bought_amount_raw, 
-    CAST(dexs.token_sold_amount_raw AS DECIMAL(38,0)) AS token_sold_amount_raw, 
+    CAST(dexs.token_bought_amount_raw AS UINT256) AS token_bought_amount_raw, 
+    CAST(dexs.token_sold_amount_raw AS UINT256) AS token_sold_amount_raw, 
```

Comment:
These fields are already UINT256 in the decoded table so no need to cast

Reply:
oke ser... will remove cast.